src/material-experimental/mdc-form-field/form-field.ts

```diff
@@ -26,7 +26,6 @@ import {
 } from '@angular/core';
 import {NgControl} from '@angular/forms';
 import {
-  FloatLabelType,
```

Comment:
@devversion can you verify this is the correct one to remove? It seems to be deprecated:
```
/**
 * Type for the available floatLabel values.
 * @deprecated Use `FloatLabelType` from `@angular/material/form-field` instead.
 * @breaking-change 11.0.0
 */
```

But the local declaration is different:
```
/** Type for the available floatLabel values. */
export type FloatLabelType = 'always' | 'auto';
```